sklearn/qda.py

```diff
@@ -1,6 +1,20 @@
 import warnings
+from .discriminant_analysis import QuadraticDiscriminantAnalysis
```

Comment:
Could we do

`from .discriminant_analysis import QuadraticDiscriminantAnalysis as _QDA`

and use that to sublcass?

class QDA(_QDA)...

(To avoid allowing import of `QuadraticDiscriminantAnalysis` from `sklearn.qda`)